modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/error-alert/ErrorAlert.js

```diff
@@ -56,33 +56,32 @@ const ErrorAlert = () => {
 	return (
 		<ClayAlert
 			displayType={apiKeyError || unknownError ? 'danger' : 'warning'}
+			title={title}
 			variant="stripe"
 		>
-			<p className="mb-0">
-				<span
-					className={
-						pageCanNotBeAudited ? 'font-weight-bold' : undefined
-					}
-				>
-					{title}{' '}
+			{pageCanNotBeAudited && (
+				<span>
+					{Liferay.Language.get(
+						'private-local-or-intranet-pages-cannot-be-audited-as-they-are-not-accessible-from-the-internet'
+					)}{' '}
 				</span>
-				{pageCanNotBeAudited && (
-					<span>
-						{Liferay.Language.get(
-							'private-local-or-intranet-pages-cannot-be-audited-as-they-are-not-accessible-from-the-internet'
-						)}{' '}
-					</span>
-				)}
-				{(apiKeyError || serverError) && (
-					<ClayButton
-						className="font-weight-bold link-primary"
-						displayType="unstyled"
-						onClick={expandInfoHandler}
-					>
-						{actionTitle}
-					</ClayButton>
-				)}
-			</p>
+			)}
+			{userHasNotPrivileges && apiKeyError && (
```

Comment:
Why did you reorder this part of the code? Should this change be in another commit? 🤔

Reply:
Susana and I decided yesterday It was incongruent to change the element's display order. [See](https://www.figma.com/file/ubR4L2j9AxVfX71hXKS4NB/LPS-134760-why-Page-Audit-is-not-working-so-I-can-fix-the-causes?node-id=44%3A55632&viewport=-1061%2C-1397%2C0.5633594393730164)
All cases follow **title** + **text**  + **error btn**
But only one followed **title** + **error btn** + **text** 

I agree it could be in another commit but it is small and does not makes any significant difference, so I add it here...